steps/apply.ts

```diff
@@ -242,6 +242,7 @@ export const completeFurtherConsiderationsTask = async (page: Page) => {
 
   const cateringRequirementsPage = await ApplyPage.initialize(page, 'Catering requirements')
```

Comment:
We changed the form so users need to supply more details when they answer 'no' instead of yes